### PR TITLE
Adds doc text for oc bugs

### PR DESCRIPTION
**Pull request by opayne1**
Adds bug doc text for `oc` bugs.

Link to docs preview- see the `oc` section.
http://file.rdu.redhat.com/opayne/RN-batch-3/release_notes/ocp-4-11-release-notes.html#ocp-4-11-bug-fixes

### CODE REVIEW

File: release_notes/ocp-4-11-release-notes.adoc [hunk 1]
```diff
@@ -1173,12 +1173,12 @@ Because non-administrator users with the `user-workload-monitoring-config-edit`
...
 With this update, the CMO now waits until deployments are deleted before recreating them, which resolves this issue.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=2069068[*BZ#2069068*])
 
-* Before this update, for user workload monitoring, if you configured external labels for metrics in Prometheus, the CMO did not correctly propagate these labels to Thanos Ruler. 
-Thus, for user-defined projects, if you queried external metrics not provided by the user workload monitoring instance of Prometheus, you would sometimes not see external labels for these metrics even though you had configured Prometheus to add them. 
+* Before this update, for user workload monitoring, if you configured external labels for metrics in Prometheus, the CMO did not correctly propagate these labels to Thanos Ruler.
+Thus, for user-defined projects, if you queried external metrics not provided by the user workload monitoring instance of Prometheus, you would sometimes not see external labels for these metrics even though you had configured Prometheus to add them.
```
Comment (tmalove) [inline release_notes/ocp-4-11-release-notes.adoc:1181]:
```suggestion
Therefore, for user-defined projects, if you queried external metrics not provided by the user workload monitoring instance of Prometheus, you would sometimes not see external labels for these metrics even though you had configured Prometheus to add them.
```

File: release_notes/ocp-4-11-release-notes.adoc [hunk 2]
```diff
@@ -1203,6 +1202,32 @@ With this update, the `tunbr` interface is now included in the list of interface
 [id="ocp-4-11-openshift-cli-bug-fixes"]
 ==== OpenShift CLI (oc)
 
+* Previously, oc catalog mirroring failed if an older, deprecated image version was used as the source. Now, the image manifest version is detected automatically and mirroring works successfully. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2049133[*BZ#2049133*])
```
Comment (ahardin-rh) [inline release_notes/ocp-4-11-release-notes.adoc:1205]: `oc`

File: release_notes/ocp-4-11-release-notes.adoc [hunk 3]
```diff
@@ -1203,6 +1202,32 @@ With this update, the `tunbr` interface is now included in the list of interface
...
+
+* Previously, it was hard to understand from the logs when fallback inspect ocured. The logs are now improved to make this more explicit. As a result, the output of `must-gather run` is much more clear. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2035717[*BZ#2035717*])
```
Comment (ahardin-rh) [inline release_notes/ocp-4-11-release-notes.adoc:1207]: occurred

File: release_notes/ocp-4-11-release-notes.adoc [hunk 4]
```diff
@@ -1203,6 +1202,32 @@ With this update, the `tunbr` interface is now included in the list of interface
...
+
+* Previously, if you ran must-gather with invalid arguments, it did not consistently report the error and instead might attempt to collect data even when that is not possible. Now if must-gather is called with invalid options, it provides useful error output. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1999891[BZ#1999891])
```
Comment (ahardin-rh) [inline release_notes/ocp-4-11-release-notes.adoc:1209]: `must-gather`

**Comment by ahardin-rh** [comment]
Just some minor comments from me. Otherwise, looks great!

**Comment by opayne1** [comment]
Updated!

**Pull request merged**